dev-docs/bidders/trustedstack.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: bidder
+title: TrustedStack
+description: Prebid TrustedStack Bidder Adaptor
+biddercode: trustedstack
+aliasCode : medianet
+gdpr_supported: true
+usp_supported: true
+coppa_supported: true
+gpp_supported: true
+media_types: banner,native,video
```

Comment:
If you support multi format, please add the two meta fields

```
multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
safeframes_ok: true/false
```

for a better vetting of your adapter